Gopkg.toml

```
@@ -22,7 +22,7 @@
 
 
 [[constraint]]
-  branch = "master"
+  version = "v0.1.0"
```

I thought we just agreed on that?

```suggestion
  version = "~0.1.0"
```

---

I'm stupid. Fixing it, sorry.